Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -236,7 +236,7 @@ protected function getFixersFromConfig($fileName, $config)
         }
 
         $fixers = array();
-        for ($i = count(self::$builtInFixers) - 1; $i >= 0; --$i) {
+        for ($i = 0, $limit = count(self::$builtInFixers); $i < $limit; ++$i) {
```

Comment:
We must not change the order of fixers. `Fixer::getFixers()` will sort fixers, so we must use sorted version.

cc @SpacePossum